src/BlogServiceProvider.php

```diff
@@ -32,6 +35,25 @@ public function configureTipoffPackage(TipoffPackage $package): void
                 \DrewRoberts\Blog\Nova\Series::class,
                 \DrewRoberts\Blog\Nova\Topic::class,
             ])
+            ->hasWebRoute('web')
+            ->hasViews()
             ->name('blog');
     }
+
+    public function bootingPackage()
+    {
+        parent::bootingPackage();
+
+        // Must happen AFTER SubstituteBindings middleware has been applied
+        app(Kernel::class)->appendToMiddlewarePriority(ResolvePage::class);
```

Comment:
Why?  You are copying things from the location package that were specific to location routing.  This should be removed.